app/src/main/java/org/wikipedia/travel/MainPlannerFragment.java

```diff
@@ -118,17 +121,17 @@ private AppCompatActivity getAppCompatActivity() {
         return (AppCompatActivity) getActivity();
     }
 
-    /*
-        Child fragment callbacks
-     */
-
     public void onLoadPage(PageTitle title, HistoryEntry entry) {
         startActivity(PageActivity.newIntentForNewTab(getContext(), entry, entry.getTitle()));
     }
 
+    /*
+        Child fragment callbacks
+     */
+
     @Override
     public void onNewTrip() {
-        CallbackTask.execute(() -> TripDbHelper.instance().createList(), new CallbackTask.DefaultCallback<Trip>(){
+        CallbackTask.execute(() -> TripDbHelper.instance().createList(), new CallbackTask.DefaultCallback <Trip>() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['<' is preceded with whitespace.](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1526985)